src/pytorch_lightning/plugins/environments/slurm_environment.py

```diff
@@ -30,9 +30,10 @@ class SLURMEnvironment(ClusterEnvironment):
             rescheduled gets determined by the owner of this plugin.
```

Comment:
Would you mind also adding the docstring for the new signal argument? Thanks :)

Reply:
Yeah completely forgot that